calendar-helper/index.js

```diff
@@ -102,9 +102,12 @@ class CalendarHelper {
 	}
 
 	getPage(m, inputDate) {
-		// I use a math clamp to check if the input date is in range
-		let dateNumber = Math.min(Math.max(inputDate, this.options.minDate), this.options.maxDate);
-		let date = new Date(dateNumber);
+		let dateNumber = this.options.minDate !== null || this.options.maxDate !== null ? Math.min(Math.max(inputDate, this.options.minDate), this.options.maxDate) : null;
```

Comment:
Please keep the comment. And also, you can avoid the `!== null`.
```suggestion
// I use a math clamp to check if the input date is in range
let dateNumber = this.options.minDate || this.options.maxDate ? Math.min(Math.max(inputDate, this.options.minDate), this.options.maxDate) : null;
```